{"uid":"dbdece2b327ac8ba","name":"Forgot Password with invalid email test","fullName":"tests.forgot_password_test.TestForgotPassword#test_invalid_email","historyId":"d2a250cc6a2f1fc6537c2941dddbd8de","time":{"start":1697677957411,"stop":1697677967923,"duration":10512},"description":"Forgot Password with invalid email address","descriptionHtml":"<p>Forgot Password with invalid email address</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5627b2043fb3 <unknown>\n#1 0x5627b1d174a7 <unknown>\n#2 0x5627b1d5edd6 <unknown>\n#3 0x5627b1d5eec1 <unknown>\n#4 0x5627b1d9c354 <unknown>\n#5 0x5627b1d8096d <unknown>\n#6 0x5627b1d99c02 <unknown>\n#7 0x5627b1d80713 <unknown>\n#8 0x5627b1d5318b <unknown>\n#9 0x5627b1d53f7e <unknown>\n#10 0x5627b20098d8 <unknown>\n#11 0x5627b200d800 <unknown>\n#12 0x5627b2017cfc <unknown>\n#13 0x5627b200e418 <unknown>\n#14 0x5627b1fdb42f <unknown>\n#15 0x5627b20324e8 <unknown>\n#16 0x5627b20326b4 <unknown>\n#17 0x5627b2043143 <unknown>\n#18 0x7f40ca894ac3 <unknown>","statusTrace":"self = <tests.forgot_password_test.TestForgotPassword object at 0x7fc66ab60050>\nexcel_reader = <utils.excel_parser.ExcelParser object at 0x7fc66a7b4750>\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"Forgot Password with invalid email address\")\n    @allure.title(\"Forgot Password with invalid email test\")\n    def test_invalid_email(self, excel_reader, json_data: dict):\n        emails = excel_reader.read_from_excel(\"Emails\")\n>       self.about_page.click_login_link()\n\ntests/forgot_password_test.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"92924d032c5a65fe91129575cbb06128\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fc66ab4de40>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5627b2043fb3 <unknown>\nE       #1 0x5627b1d174a7 <unknown>\nE       #2 0x5627b1d5edd6 <unknown>\nE       #3 0x5627b1d5eec1 <unknown>\nE       #4 0x5627b1d9c354 <unknown>\nE       #5 0x5627b1d8096d <unknown>\nE       #6 0x5627b1d99c02 <unknown>\nE       #7 0x5627b1d80713 <unknown>\nE       #8 0x5627b1d5318b <unknown>\nE       #9 0x5627b1d53f7e <unknown>\nE       #10 0x5627b20098d8 <unknown>\nE       #11 0x5627b200d800 <unknown>\nE       #12 0x5627b2017cfc <unknown>\nE       #13 0x5627b200e418 <unknown>\nE       #14 0x5627b1fdb42f <unknown>\nE       #15 0x5627b20324e8 <unknown>\nE       #16 0x5627b20326b4 <unknown>\nE       #17 0x5627b2043143 <unknown>\nE       #18 0x7f40ca894ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1697677764973,"stop":1697677764973,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1697677764973,"stop":1697677764973,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"json_data","time":{"start":1697677764973,"stop":1697677764973,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"description":"Forgot Password with invalid email address","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5627b2043fb3 <unknown>\n#1 0x5627b1d174a7 <unknown>\n#2 0x5627b1d5edd6 <unknown>\n#3 0x5627b1d5eec1 <unknown>\n#4 0x5627b1d9c354 <unknown>\n#5 0x5627b1d8096d <unknown>\n#6 0x5627b1d99c02 <unknown>\n#7 0x5627b1d80713 <unknown>\n#8 0x5627b1d5318b <unknown>\n#9 0x5627b1d53f7e <unknown>\n#10 0x5627b20098d8 <unknown>\n#11 0x5627b200d800 <unknown>\n#12 0x5627b2017cfc <unknown>\n#13 0x5627b200e418 <unknown>\n#14 0x5627b1fdb42f <unknown>\n#15 0x5627b20324e8 <unknown>\n#16 0x5627b20326b4 <unknown>\n#17 0x5627b2043143 <unknown>\n#18 0x7f40ca894ac3 <unknown>","statusTrace":"self = <tests.forgot_password_test.TestForgotPassword object at 0x7fc66ab60050>\nexcel_reader = <utils.excel_parser.ExcelParser object at 0x7fc66a7b4750>\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"Forgot Password with invalid email address\")\n    @allure.title(\"Forgot Password with invalid email test\")\n    def test_invalid_email(self, excel_reader, json_data: dict):\n        emails = excel_reader.read_from_excel(\"Emails\")\n>       self.about_page.click_login_link()\n\ntests/forgot_password_test.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"92924d032c5a65fe91129575cbb06128\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fc66ab4de40>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5627b2043fb3 <unknown>\nE       #1 0x5627b1d174a7 <unknown>\nE       #2 0x5627b1d5edd6 <unknown>\nE       #3 0x5627b1d5eec1 <unknown>\nE       #4 0x5627b1d9c354 <unknown>\nE       #5 0x5627b1d8096d <unknown>\nE       #6 0x5627b1d99c02 <unknown>\nE       #7 0x5627b1d80713 <unknown>\nE       #8 0x5627b1d5318b <unknown>\nE       #9 0x5627b1d53f7e <unknown>\nE       #10 0x5627b20098d8 <unknown>\nE       #11 0x5627b200d800 <unknown>\nE       #12 0x5627b2017cfc <unknown>\nE       #13 0x5627b200e418 <unknown>\nE       #14 0x5627b1fdb42f <unknown>\nE       #15 0x5627b20324e8 <unknown>\nE       #16 0x5627b20326b4 <unknown>\nE       #17 0x5627b2043143 <unknown>\nE       #18 0x7f40ca894ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1697677957449,"stop":1697677967922,"duration":10473},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5627b2043fb3 <unknown>\n#1 0x5627b1d174a7 <unknown>\n#2 0x5627b1d5edd6 <unknown>\n#3 0x5627b1d5eec1 <unknown>\n#4 0x5627b1d9c354 <unknown>\n#5 0x5627b1d8096d <unknown>\n#6 0x5627b1d99c02 <unknown>\n#7 0x5627b1d80713 <unknown>\n#8 0x5627b1d5318b <unknown>\n#9 0x5627b1d53f7e <unknown>\n#10 0x5627b20098d8 <unknown>\n#11 0x5627b200d800 <unknown>\n#12 0x5627b2017cfc <unknown>\n#13 0x5627b200e418 <unknown>\n#14 0x5627b1fdb42f <unknown>\n#15 0x5627b20324e8 <unknown>\n#16 0x5627b20326b4 <unknown>\n#17 0x5627b2043143 <unknown>\n#18 0x7f40ca894ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[{"uid":"7bb8ddfb81169263","name":"public ip address","source":"7bb8ddfb81169263.txt","type":"text/plain","size":13},{"uid":"8bc990c4848c2ee9","name":"Cookies","source":"8bc990c4848c2ee9.json","type":"application/json","size":2018},{"uid":"50738deae50a4f9b","name":"Session Storage","source":"50738deae50a4f9b.json","type":"application/json","size":333},{"uid":"3e8f164cdea894ef","name":"Local Storage","source":"3e8f164cdea894ef.json","type":"application/json","size":208},{"uid":"6ebc3817a90e611c","name":"Console Logs","source":"6ebc3817a90e611c.json","type":"application/json","size":2},{"uid":"b247681df14c4b8a","name":"Full Page Screenshot","source":"b247681df14c4b8a.png","type":"image/png","size":714221},{"uid":"a61bdcd2e2633264","name":"URL","source":"a61bdcd2e2633264.uri","type":"text/uri-list","size":23},{"uid":"f42e40465e642483","name":"Network Logs","source":"f42e40465e642483.json","type":"application/json","size":23089}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true},"afterStages":[],"labels":[{"name":"epic","value":"Security"},{"name":"parentSuite","value":"Custom parent suite"},{"name":"severity","value":"critical"},{"name":"story","value":"Forgot Password Feature's Functionality"},{"name":"subSuite","value":"Custom sub suite"},{"name":"suite","value":"Custom suite"},{"name":"tag","value":"security"},{"name":"host","value":"fv-az797-587"},{"name":"thread","value":"2248-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.forgot_password_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security"]},"source":"dbdece2b327ac8ba.json","parameterValues":[]}